drivers/misc/rwbuffer.c

```diff
@@ -1210,7 +1212,7 @@ int rwb_mediaremoved(FAR struct rwbuffer_s *rwb)
     }
 #endif
 
-  return OK;
+  return ret;
```

Comment:
```suggestion
  return OK;
```

Reply:
Styling in this file applies both therefore was not sure which way to apply.  e.g. refer to rwb_write().  If required can change to style of function below as opposed to function above.